example/dev-env-setup/roles/snmp/tasks/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+- name: install snmp
+  apt: pkg=snmp state=installed
```

Comment:
A general question, I didn't see any version control on tools like snmp, ipmitool, mongo, rabbitmq, isc-dhcp-server and so on.
You should add that.